snippets/csharp/programming-guide/nullable-types/NullableTypesUsage.cs

```diff
@@ -140,5 +141,21 @@ private static void ComparisonOperators()
             // num1 == num2 is true if the value of each is null
             // </Snippet8>
         }
+
+        private static void BoxingAndUnboxing()
+        {
+            // <Snippet9>
+            int a = 41;
+            object aBoxed = a;
+            int? aNullable = (int?)aBoxed;
+            Console.WriteLine($"Value of aNullable: {aNullable}");
+
+            object aNullableBoxed = aNullable;
+            if (aNullableBoxed is int aValue)
+            {
+                Console.WriteLine($"aNullableBoxed is boxed int: {aValue}");
+            }
+            // </Snippet9>
```

Comment:
Could you add output from the example?

Reply:
@rpetrusha I've updated the snippet with its output.